magicctapipe/scripts/lst1_magic/lst_m1_m2_cloud_correction.py

```diff
@@ -0,0 +1,417 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+This script corrects LST-1 and Magic data for the cloud affection.
+
+Usage:
+$ python lst_m1_m2_cloud_correction.py
+--input-file dl1_stereo/dl1_LST-1_MAGIC.Run03265.0040.h5
+(--output-dir dl1_corrected)
+"""
+import argparse
+import logging
+
+import astropy.units as u
+import numpy as np
+import pandas as pd
+from astropy.coordinates import AltAz, SkyCoord
+from ctapipe.coordinates import TelescopeFrame
+from ctapipe.image import (
+    concentration_parameters,
+    hillas_parameters,
+    leakage_parameters,
+    timing_parameters,
+)
+from ctapipe.instrument import SubarrayDescription
+from ctapipe.io import read_table
+from lstchain.reco import disp
+from lstchain.reco.utils import sky_to_camera
+
+from magicctapipe.io import save_pandas_data_in_table
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.StreamHandler())
+logger.setLevel(logging.INFO)
+
+
+def model0(imp, h, zd):
+    """
+    Calculates distance
+
+    Parameters
+    ----------
+    imp : float
+        Impact
+    h : float
+        Height
+    zd : float
+        Zenith distance
```

Comment:
if the values are floats you should specify units

Reply:
Done